pythonforandroid/toolchain.py

```diff
@@ -304,7 +305,7 @@ def __init__(self):
                   '(default: {})'.format(default_storage_dir)))
 
         generic_parser.add_argument(
-            '--arch', help='The archs to build for, separated by commas.',
+            '--arch', help='The arch to build for.',
```

Comment:
I'm wondering, are we breaking our CLI API here? Wasn't the arch split via `split_argument_list()` later?

Reply:
Yes, a bit, but I did this because as far as I can tell passing multiple archs won't actually work right now. I'm trying to avoid removing the internal flexibility (hence keeping lists of archs even though we only handle one), but it was simplest to make the cli match what we can actually support for now. I'll confirm this before finalising the PR.

Reply:
Yes that makes complete sense, thanks. I was also confused seeing it was handling multiple archs from the CLI.